config/settings.yml

```diff
@@ -65,7 +65,7 @@ authorization_workgroup_names:
 
 # See https://www.oclc.org/developer/develop/web-services/fast-api/assign-fast.en.html for more information
 autocomplete_lookup:
-  url: http://fast.oclc.org/fastIndex/select
+  url: http://fast.oclc.org/searchfast/fastsuggest
```

Comment:
@amyehodge @arcadiafalcone Here's the new FAST API suggested by #3581 (https://www.oclc.org/developer/api/oclc-apis/fast-api/assign-fast.en.html). It's different from the former API, though. See my other comments for more.